server/helpers/dbHelper.js

```diff
@@ -110,8 +110,11 @@ module.exports = (db) => {
       .query(
         `
         INSERT INTO blacklists (user_id, website_id)
-        VALUES ($1, $2)
+        SELECT $1, $2
```

Comment:
Not sure why lines 113-115 are here. With the unique constraint you added you can just replace these lines with `VALUES ($1, $2)` and the db will return an error if you try to insert duplicate values.

Reply:
Yea this Query can revert back to what it was before. I was just testing to see diff methods of achieving the same goal. It's essentially better to have the constraint on the table instead of in queries.